app/src/main/java/com/android/sample/ui/authentication/SignUp.kt

```diff
@@ -198,13 +179,7 @@ fun SignUpScreen(navigationActions: NavigationActions) {
                 Text(
                     modifier = Modifier.testTag("loginText"),
                     text = "Already have an account ?",
-                    style =
-                        TextStyle(
-                            fontSize = 14.sp,
-                            fontFamily = FontFamily(Font(R.font.roboto)),
-                            fontWeight = FontWeight(500),
-                            color = PrimaryPurple,
-                            letterSpacing = 0.14.sp))
+                    style = TypographyBeatLink.bodyMedium)
```

Comment:
Consider utilizing MaterialTheme.typography to further centralize theme control and maintain consistency throughout the application.

```suggestion
                    style = MaterialTheme.typography.bodyMedium)
```